### PR TITLE
PR for #36 - Implemented Package API for iOS

### PR DESCRIPTION
**Pull request by jamesmcroft**
This PR includes changes to bring the Package API to iOS.

### CODE REVIEW

File: XPlat.ApplicationModel/Package.iOS.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        public IStorageFolder InstalledLocation => this.installedLocation ?? (this.installedLocation = new StorageFolder(this.Originator.BundlePath));
+
+        /// <summary>Gets the packages on which the current package depends.</summary>
+        public IReadOnlyList<IPackage> Dependencies => new List<IPackage>();
+
+        /// <summary>Gets the display name of the package.</summary>
+        public string DisplayName => this.Originator?.ObjectForInfoDictionary("CFBundleDisplayName")?.ToString();
+
+        /// <summary>Gets the logo of the package.</summary>
+        public Uri Logo => null;
```
Comment (tom-made) [inline XPlat.ApplicationModel/Package.iOS.cs]: Why doesn't this return anything?

Comment (jamesmcroft) [reply XPlat.ApplicationModel/Package.iOS.cs]: So I couldn't work out or find much around documentation on how to get the path to the logo. I thought I would leave this, for now, unimplemented until potentially I find a result on this.

Are you happy with that?

Comment (tom-made) [reply XPlat.ApplicationModel/Package.iOS.cs]: That's not a problem, as long as this is documented

Comment (jamesmcroft) [reply XPlat.ApplicationModel/Package.iOS.cs]: I will add some text in the GitBooks docs for this 👍

Comment (tom-made) [reply XPlat.ApplicationModel/Package.iOS.cs]: Awesome, thanks!

**Pull request merged**